SR2020/2020-02-19-inactive-team-prompt.md

```diff
@@ -0,0 +1,41 @@
+---
+to: Teams deemed 'inactive'
+subject: Lack of activity in Student Robotics
+---
+
+Hello There!
+
+Our records show you've not been especially active on the IDE or Forum recently, and we want to offer our support!
```

Comment:
How are we planning to get the forum data?
We probably can get that, though it's not something we've got an existing script for. Would be a good one to add to the toolbox though.

Reply:
ah, I'd forgotten we don't track forum activity, will amend

Reply:
We could potentially write a phpBB extension for this. I'd potentially be willing to give it a shot depending on how soon you need it.

Reply:
The IDE is a far better and more useful metric right now for team participation. In hindsight, we have a lot of teams which never use the IDE and yet are very compitent and active teams, who likely wouldn't need chasing.

That being said, i'm sure there are better metrics than simply the IDE / number of competitors